docs/plugin-cerberus.md

```diff
@@ -3,8 +3,6 @@
 [Pinery](http://github.com/oicr-gsi/pinery) and
 [Vidarr](https://github.com/oicr-gsi/vidarr).
 
-Deploying this plugin requires the [gsi-common](../plugin-gsi-common/README.md) plugin be deployed as well.
```

Comment:
I believe this is still a requirement?

Reply:
Yes, but the module system takes care of this, so there's no need to specify it.